client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTabStyles.ts

```diff
@@ -0,0 +1,77 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+    tabInitialContainer: {
+        paddingTop: '2vh',
+        height: '64vh'
+    },
+    PersonalInfoFieldContainer: {
+        display: 'flex',
+        alignItems: 'center'
+    },
+    personalInfoLastFieldContainer: {
+        display: 'flex',
```

Comment:
you have this flex and alignItems in the previous class why not just do className={personalInfoLastFieldContainer + PersonalInfoFieldContainer} or just add padding to PersonalInfoFieldContainer?